examples/blueprint-construct/index.ts

```diff
@@ -20,7 +20,7 @@ export interface BlueprintConstructProps {
 export default class BlueprintConstruct {
     constructor(scope: Construct, props: cdk.StackProps) {
 
-        HelmAddOn.validateHelmVersions = true;
+        //HelmAddOn.validateHelmVersions = true;
```

Comment:
please fix the three github actions warning below like 'teams' is assigned a value but never used

Reply:
Should I revert this file back to what it was before since I was using it for testing purposes?

Reply:
Yes, please revert.